Content.Server/GameObjects/Components/Power/PowerNetComponents/BatteryStorageComponent.cs

```diff
@@ -31,21 +35,26 @@ public override void ExposeData(ObjectSerializer serializer)
         public override void Initialize()
         {
             base.Initialize();
-
-            _battery = Owner.EnsureComponent<BatteryComponent>();
-            Consumer = Owner.EnsureComponent<PowerConsumerComponent>();
+            _battery = Owner.EnsureComponentWarn<BatteryComponent>();
+            _consumer = Owner.EnsureComponentWarn<PowerConsumerComponent>();
```

Comment:
nitpick: you are assigning this twice because compdeps do it in the background. call the function but dont use the return value imo